import
	io

main fut exit-code(args arr str) summon
	0 resolved
